.github/workflows/jacoco_report.yml

```diff
@@ -4,6 +4,8 @@ on:
   pull_request:
     branches:
       - master
+    paths:
+      - "airbyte-integrations/connectors/source-postgres/**"
```

Comment:
You got all the dependencies in base-java and the abstract classes

Reply:
I was thinking about that as well, but it won't affect source-postgres unless someone publishes it. And also Jacoco is scanning only for changes in that module.